### ISSUE TITLE
miss assets file after update online

### ISSUE THREAD

**Issue by iwater** [Microsoft/react-native-code-push#79]
I know there is a issue about update assets，but this issue is after update a jsbundle online, the origin assets ship with ipa can't show anymore, react-native version 0.15.0, code-push version 1.3.1-beta

**Comment by lostintangent** [comment]
Hey @iwater, I believe the assets system expects them to be located in an "assets" folder next to the JS bundle file on disk. Since we write the updated JS bundle to a new directory on disk, but don't copy over the assets from the binary, then when the RN runtime loads the new bundle, it won't find the original assets.

We are working on asset support right now and will release it this week. Stay tuned!

**Comment by iwater** [comment]
thank you for your hard work

**Comment by oney** [comment]
Quick solution is
Place your images to `images` folder, drag to Xcode project. Uncheck `Copy items if needed` and select `Create folder references`. Then, the images folder will be added to project and show blue color.
![](http://i.imgur.com/RJHgLXP.png)
![](http://i.imgur.com/yODhUcI.png)
Change your images object to uri
```javascript
require('../images/mypic.png')
{uri: './images/mypic.png'} // <- change to
```

**Comment by lostintangent** [comment]
Hey @oney, would this solution lose out on the ability to have multiple images for different pixel densities (e.g. foo.png and foo@2x.png) and have the right one get picked up at runtime? I had thought to get that behavior you would need to use xcasset files if you weren't going to use the Reacf Native asset loader.

**Comment by oney** [comment]
Yes, you are right.
Put image to xcasset and using `{uri: 'mypic.png'}` will get the image?

**Comment by lostintangent** [comment]
Hey @iwater, this issue was addressed with #82, so I'm going to close this issue. Let us know if you get a chance to try it out!
